recipes-core/mongoose/mongoose.inc

```diff
@@ -20,6 +20,10 @@ S = "${WORKDIR}/git"
 CFLAGS_prepend = "-Wno-format-truncation -fPIC "
 EXTRA_OEMAKE = "'CC=${CC}' 'AR=${AR}' 'CFLAGS=${CFLAGS}' 'DESTDIR=${D}' 'PREFIX=/usr'"
 
+FILES_SOLIBSDEV = ""
+FILES_${PN} += "${libdir}/"
+INSANE_SKIP_${PN} = "dev-so"
```

Comment:
I'm not too happy about the `INSANE_SKIP` here. I can't fully recall: Did we validate the necessity, output and potential issues?

Reply:
Necessity is there, unfortunately, and output works - potential issues I am not certain about.